README.md

```diff
@@ -51,6 +51,7 @@ This repository includes files with plain SQL that can be used to recreate a dat
 - **[Animal-table]**
 - **[Update-table]**
 - **[Query-animal]**
+- **[Multiple-table]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly remove the following links as they point to subsections that have been removed making them broken links
![image](https://github.com/Lawmsangi/vet-clinic-database/assets/81932478/7032ddd9-9835-40ac-ae88-6db427e696f1)